### PR TITLE
[MFTF] AdminAddNewDefaultBillingShippingCustomerAddressTest refactoring

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
The test is refactored according to the best practices

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B, Semantic Version Checker

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminClickEditLinkForDefaultBillingAddressActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <actionGroup name="AdminClickEditLinkForDefaultBillingAddressActionGroup">
+        <annotations>
+            <description>Clicks the Edit link for Default Billing Address.</description>
+        </annotations>
+
+        <click selector="{{AdminCustomerAddressesDefaultBillingSection.editButton}}" stepKey="clickEditDefaultBillingAddress"/>
+        <waitForPageLoad stepKey="waitForFormToLoad"/>
+    </actionGroup>
+</actionGroups>
+
```
Comment (Den4ik) [inline app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminClickEditLinkForDefaultBillingAddressActionGroup.xml]: Please add new line at the end of file

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminFillAndSaveCustomerlAddressInformationActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <click selector="{{AdminCustomerAddressesSection.country}}" stepKey="clickCountryToOpenListOfCountries"/>
+        <click selector="{{AdminCustomerAddressesSection.countryId(address.country_id)}}" stepKey="fillCountry"/>
+        <fillField userInput="{{address.postcode}}" selector="{{AdminCustomerAddressesSection.zip}}" stepKey="fillPostcode"/>
+        <fillField userInput="{{address.telephone}}" selector="{{AdminCustomerAddressesSection.phoneNumber}}" stepKey="fillTelephone"/>
+        <click selector="{{AdminCustomerAddressesSection.region}}" stepKey="clickRegionToOpenListOfRegions"/>
+        <click selector="{{AdminCustomerAddressesSection.regionId(address.state)}}" stepKey="fillRegion"/>
+        <click selector="{{AdminCustomerAddressesSection.saveAddress}}" stepKey="clickSaveCustomerAddressOnAddUpdateAddressForm"/>
+        <waitForPageLoad stepKey="waitForNewAddressIsCreated"/>
+    </actionGroup>
+</actionGroups>
```
Comment (Den4ik) [inline app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminFillAndSaveCustomerlAddressInformationActionGroup.xml]: Please add new line at the end of file

**Review by Den4ik — CHANGES_REQUESTED** [review]
@AnnaAPak Thanks for work on tests.
Please look at my comments

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B, Semantic Version Checker

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminAssertCustomerDefaultShippingAddressAgainstEntityActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertCustomerDefaultShippingAddressAgainstEntityActionGroup">
+        <annotations>
+            <description>Asserts that Customer's Default Billing Address matches the specified Address entity</description>
```
Comment (eduard13) [inline app/code/Magento/Customer/Test/Mftf/ActionGroup/AdminAssertCustomerDefaultShippingAddressAgainstEntityActionGroup.xml]:
```suggestion
            <description>Asserts that Customer's Default Shipping Address matches the specified Address entity</description>
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Please check the above suggestion

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B, Semantic Version Checker

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
It seems that all the review comment's have been taken care and rectified, so moving it further.

Thank you!

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-9251](https://jira.corp.magento.com/browse/ENGCOM-9251) has been created to process this Pull Request
:eight_spoked_asterisk: @eduard13, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by Den4ik — APPROVED** [review]

**Pull request merged**